GurkiratHans/homework1.py

```diff
@@ -0,0 +1,143 @@
+# 1. Create a program that prints your name on screen.
+print('Gurkirat Hans')
+
+# 2. Create two variables, name and age, and assign your name and age to these variables. Then, print a message saying "My name is [name], and I am [age] years old."
+name = 'Gurkirat Hans'
+age = '29'
```

Comment:
best not write age as a string. I know why you did it, but perhaps later in the code you would want to modify the age. You can't do this as a string.

Reply:
Changing